### DISCUSSION TITLE
High school student

### DISCUSSION THREAD

**Discussion by SebastienXia** [community/community#110835] [GitHub Education]
### Hi everyone,

Hello everyone,

I'm a **_high school student_** and I was wondering if high school students can apply for the _Student Developer Pack_ ?

![provide-github-student-developer-pack-account](https://github.com/community/community/assets/105004912/8bd3331f-6d4c-434b-95c5-860a30bb2238)

**Comment by thekarthikr** [comment]
Yes, high school students are eligible to apply for the GitHub Student Developer Pack. Simply follow the application process on the GitHub Education website to apply for the pack. You need a student ID card with the academic year, or other certifications related to your academic status.

**Comment by notSaqlain** [comment]
Absolutely! High school students are welcome to apply for the GitHub Student Developer Pack. The program is open to anyone currently enrolled in a degree or diploma granting course of study, including:

-> High school
-> Secondary school
-> College
-> University
-> Homeschool
-> Similar educational institution

Here are the requirements to apply for the Student Developer Pack:

**_Be currently enrolled:_** You'll need to provide proof of your enrollment, such as a school email address or a scan of your student ID.
**_Have a GitHub account:_** Create a free account on https://github.com/ if you haven't already.
**_Be at least 13 years old:_** This is the minimum age requirement for using GitHub.
Once you meet these requirements, you can head over to the GitHub Student Developer Pack application page: https://education.github.com/.

If you have any further questions or encounter any issues during the application process, don't hesitate to check out the GitHub documentation for more information and troubleshooting tips: https://education.github.com/students.

[mark as complete and close the discussion if this helps, Thanks!]
